pkgs/top-level/all-packages.nix

```diff
@@ -18882,8 +18882,6 @@ with pkgs;
 
   black = with python3Packages; toPythonApplication black;
 
-  blackfire = callPackage ../development/tools/misc/blackfire { };
```

Comment:
It looks that the package can be migrated automatically, so if I don't misread the [manual migration guidelines](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#manual-migration-guidelines), the migration here is not really encouraged.

Reply:
I didn't know about that info, thanks!

Note that point 2 says that manual migration is fine if the code is worked on anyway (like a package update).